src/main/java/com/google/cloud/tools/appengine/v2/internal/cloudsdk/CloudSdkV2.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.v2.internal.cloudsdk;
+
+import com.google.cloud.tools.appengine.api.AppEngineException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.logging.Logger;
+
+/**
+ * Cloud SDK CLI wrapper.
+ *
+ * TODO : Need to revisit usage of path resolver
+ */
+public class CloudSdkV2 {
```

Comment:
I don't know exactly what to do here, when making everything provided this is simply a data class. There might need to be a little more refactoring around this.